bokun.yaml

```diff
@@ -5537,413 +5734,535 @@ definitions:
       attention:
         type: string
         description: 'Text describing any points that need special attention for the passengers. May contain HTML.'
-      baseLanguage:
-        type: string
-        description: 'The language that the product description is written in. There may also be translations available.'
-      bookableExtras:
+      locationCode:
+        $ref: '#/definitions/LocationCodeDto'
+        description: 'The city that represents the main location for this activity. Specified as a location from the United Nations Code for Trade and Transport Locations (UN/LOCODE).'
+      googlePlace:
+        type: object
+        properties:
+          country:
+            type: string
+          countryCode:
+            type: string
+          city:
+            type: string
+          cityCode:
+            type: string
+          geoLocationCenter:
+            type: object
+            description: 'Represents a point in geographical coordinates: latitude and longitude.'
+            properties:
+              lat:
+                type: number
+                format: double
+              lng:
+                type: number
+                format: double
+      resourceSlots:
         type: array
-        description: 'The list of extras available for this product. Note that not all of these extras may be available for all bookable items. For example, different rates may offer different extras.'
         items:
-          $ref: '#/definitions/BookableExtraDto'
-      bookingCutoffDays:
+          type: object
+          properties:
+            id:
+              type: integer
+              format: int64
+            count:
+              type: integer
+              format: int32
+        description: 'Not working.'
+      bookingCutoffMinutes:
         type: integer
         format: int32
-        description: 'The number of days before the activity when we stop accepting bookings.'
+        description: 'The number of minutes before the activity when we stop accepting bookings.'
       bookingCutoffHours:
         type: integer
         format: int32
         description: 'The number of hours before the activity when we stop accepting bookings.'
-      bookingCutoffMinutes:
+      bookingCutoffDays:
         type: integer
         format: int32
-        description: 'The number of minutes before the activity when we stop accepting bookings.'
+        description: 'The number of days before the activity when we stop accepting bookings.'
       bookingCutoffWeeks:
         type: integer
         format: int32
         description: 'The number of weeks before the activity when we stop accepting bookings.'
-      box:
-        type: boolean
-        description: 'Specifies whether this is a boxed version of another product.'
+      requestDeadlineMinutes:
+        type: integer
+        format: int32
+        description: 'The number of minutes allocated to confirm or reject booking by activity supplier.'
```

Comment:
Same as for cutoff - that all 4 fields work together if specified.

Reply:
done